src/web/js/beforePyret.js

```diff
@@ -194,7 +197,8 @@ $(function() {
         "Esc Right": "goForwardSexp",
         "Alt-Right": "goForwardSexp",
         "Ctrl-Left": "goBackwardToken",
-        "Ctrl-Right": "goForwardToken"
+        "Ctrl-Right": "goForwardToken",
+        [`${modifier}-/`]: "toggleComment",
```

Comment:
Ah OK no change for `{modifier}` here, which is least surprise. Thanks!

Reply:
Yup - wanted to retain the same functionality for all other commands (on a mac, Cmd-Right/Left) are still start/end of line.